tests/RandomTest.php

```diff
@@ -506,7 +508,7 @@ public function testDuplicateSaveNew()
         $db = new Persistence\Sql($this->db->connection);
         $m = new Model_Rate($db);
 
-        $m->load(1)->duplicate()->save();
+        (clone $m)->load(1)->duplicate()->save();
```

Comment:
why clone?

Reply:
because later needed for a full export (which iterates over model and can not longer over entity (as entity = model restricted with one ID, thus iterating over one model does not make sense)) ;-)